bigquery_etl/cli/metadata.py

```diff
@@ -126,3 +128,51 @@ def publish(name: str, sql_dir: Optional[str], project_id: Optional[str]) -> Non
             print("No metadata file for: {}.{}.{}".format(project, dataset, table))
 
     return None
+
+
+@metadata.command(
+    help="""
+    Deprecate BigQuery table by updating metadata.yaml file.
+    Deletion date is by default 3 months from current date if not provided.
+
+    Example:
+     ./bqetl metadata deprecate ga_derived.downloads_with_attribution_v2 --deletion_date=2024-03-02
+    """
+)
+@click.argument("name")
+@project_id_option(
+    ConfigLoader.get("default", "project", fallback="moz-fx-data-shared-prod")
+)
+@sql_dir_option
+@click.option(
+    "--deletion_date",
+    "--deletion-date",
+    "-d",
```

Comment:
For cases like this where it isn't a common parameter IMO it'd be preferable not to have a short option, which are more cryptic.